cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -223,27 +223,24 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
         }
 
         val includedLicenseCategories = globalOptionsForSubcommands.config.downloader.includedLicenseCategories
+        if (includedLicenseCategories.isNotEmpty() && licenseClassificationsFile.isFile) {
+            val licenseCategorizations = licenseClassificationsFile.readValue<LicenseClassifications>().categorizations
+            val licenseInfoResolver = ortResult.createLicenseInfoResolver()
```

Comment:
Unrelated to this commit, but this makes me realize that we do not provide the correct package configuration provider here which can lead to wrong results. It should be initialized like in `EvaluatorCommand`, just in case you want to fix this. Otherwise let me know, then I will make a separate PR later.